Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/azure-policy-deploy.yaml

```diff
@@ -46,17 +46,19 @@ spec:
           value: Microsoft.Kubernetes/connectedClusters
         - name: DATAPLANE_ENDPOINT
           value: https://gov-prod-policy-data.trafficmanager.net
-        - name: REFRESH_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
-        - name: AUDIT_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
-        - name: TELEMETRY_HEARTBEAT_INTERVAL
-          value: 5m   # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h"
         - name: CLIENT_SECRET
           valueFrom:
             secretKeyRef:
               name: azure-policy
               key: client-secret
+        - name: POD_NAME
```

Comment:
Need to remove INTERVALs in Azure Policy helm chart that Ruchika has currently for AKS. Can you please follow up with Ruchika on this?

Reply:
Is there any rush to remove the intervals from AKS? The reconcile loop ensures the intervals can't be changed by the user. If there's no rush, can we put the change in with Gatekeeper v3 changes?

Reply:
Fine with me.